### PR TITLE
Support for accessing IPD in scripts

### PR DESCRIPTION
**Pull request by jherico**
One of the most useful tools for debugging HMD setups is a color cube of exactly the same width as the viewers interpupillary distance.  This modification makes it possible to create one via JS.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: ac0258516795f1d134f9e894340e8a5ed4ad7ea1 available at: https://gist.github.com/8bc440be8f3f5b4576d5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7b1c04512a34258c7c8ffccb80c9f4a416424160 available at: https://gist.github.com/8bc440be8f3f5b4576d5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.dmg

File: examples/example/hmd/colorCube.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        userData: JSON.stringify(that.USER_DATA)
+    });
+}
+
+var ColorCube = new ColorCube();
+ColorCube.clear();
+ColorCube.create();
+
+
+var ids = Entities.findEntities(MyAvatar.position, 50);
```
Comment (ZappoMan) [inline examples/example/hmd/colorCube.js]: why does this code from line 45-52 exist? doesn't the ColorCube.clear() do this??

File: libraries/display-plugins/src/display-plugins/DisplayPlugin.h [hunk 2]
```diff
@@ -121,6 +121,10 @@ class DisplayPlugin : public Plugin {
         static const glm::mat4 pose; return pose;
     }
 
+    virtual float getIPD() const {
```
Comment (ZappoMan) [inline libraries/display-plugins/src/display-plugins/DisplayPlugin.h]: coding standard: this should be a single line

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9b13aac37db162a126b7ae1bcf51fa79d50959d6 available at: https://gist.github.com/8bc440be8f3f5b4576d5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5924.dmg

**Pull request merged**